api/src/client/dynamics/license-status/DynamicsLicenseApplicationIdClient.ts

```diff
@@ -67,6 +67,7 @@ const licenseTypeToLicenseId: Record<string, string> = {
   "Public Movers and Warehousemen": "19391a3f-53df-eb11-bacb-001dd8028561",
   "Home Improvement Contractor": "7a391a3f-53df-eb11-bacb-001dd8028561",
   "Health Care Services": "3ac8456a-53df-eb11-bacb-001dd8028561",
+  "Health Club Services": "af8e3564-53df-eb11-bacb-001dd8028561",
```

Comment:
The key here is `Health Club Services` but the licenseType is `Health Club` in the industry file. I don't think the search is enabled

Reply:
The search is enabled. I was able to retrieve license status for both expired and active licenses. I do wonder whether we should have consistent nomenclature  between the back and front end.